examples/nlmchem.py

```diff
@@ -0,0 +1,342 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and Samuele Garda.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This script was originally authored by Jason Fries"""
```

Comment:
Hi @sg-wbi Did you and Jason worked on this together? FYI So far, we have not all other examples files doesn't include an authorship information at the beginning of the file.

Reply:
I suggest that we leave this one out, to be consistent with other example files

Reply:
Sure! I think I added it to point out that I modified `examples/bc5cdr.py`